app/controllers/admin/settings_controller.rb

```diff
@@ -59,8 +59,14 @@ def settings_by_group(group)
         [all_settings["social.facebook"]] + [all_settings["social.twitter"]] + [all_settings["social.google"]]
       when "advanced"
         [all_settings["advanced.auth"]] + [all_settings["advanced.tracking"]]
+      when "regional"
+        [all_settings["regional.default_locale"]] + [valid_regional_available_locales(all_settings)] + [all_settings["regional.time_zone"]]
       else
         all_settings[group]
       end
     end
+
+    def valid_regional_available_locales(all_settings)
+      all_settings["regional.available_locale"].select { |s| s.key.rpartition(".").last.to_sym != I18n.default_locale }
```

Comment:
```suggestion
      all_settings["regional.available_locales"].select { |s| s.key.rpartition(".").last.to_sym != I18n.default_locale }
```
🤔

Reply:
I put it in singular because the key of each available locale follows the following structure:
`regional.available_locale.es`
`regional.available_locale.en`
`regional.available_locale.xx`
and I think it looks better than:
`regional.available_locales.xx`